.circleci/config.yml

```diff
@@ -53,7 +54,6 @@ jobs:
             apt update
             apt install -y build-essential zlib1g-dev uuid-dev libdigest-sha-perl libelf-dev bc bzip2 bison flex git gnupg gawk iasl m4 nasm patch python python2 python3 wget gnat cpio ccache pkg-config cmake libusb-1.0-0-dev autoconf texinfo ncurses-dev doxygen graphviz udev libudev1 libudev-dev automake libtool rsync innoextract sudo imagemagick libncurses5-dev
       - checkout
```

Comment:
@JonathonHall-Purism on fixing CircleCI to get tags, this is where that could happen?

Reply:
@JonathonHall-Purism issue was that my tlaurion repo included a "Release" tag for 5.0.1 that never happened. Used it as a trace to test reproducible builds a while back. Removed it.